tests/dummy/app/templates/share-discover-page.hbs

```diff
@@ -0,0 +1,33 @@
+{{title 'Search'}}
+
+{{discover-page
+    clearFiltersButton="Clear filters"
```

Comment:
I was providing more options for customization, but have since removed a few of them.  This is not so many configuration options given that the component builds an entire page. Especially since half of the properties passed in are the individual query parameters so when they are updated in the component, they will be reflected in the URL.